katran/lib/bpf/handle_icmp.h

```diff
@@ -118,8 +118,8 @@ static inline int send_icmp4_too_big(struct xdp_md *xdp) {
   if (bpf_xdp_adjust_head(xdp, 0 - headroom)) {
     return XDP_DROP;
   }
-  void *data = xdp->data;
-  void *data_end = xdp->data_end;
+  void *data = (void *)(long)xdp->data;
```

Comment:
Can you use `uintptr_t` just to be pedantically correct? It's in linux/types.h so it should be fine.

Reply:
uintptr_t is not in uapi

Reply:
Lame. Then it's fine.